src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -614,4 +622,12 @@ private void rotateLeftClicked()
         rotation = (rotation + ROTATE_LEFT) % POSSIBLE_ROTATIONS;
         updateRotation(rotation);
     }
+
+    /**
+     * Rotate the structure counter clockwise.
+     */
+    private void mirror()
```

Comment:
I think the idea here is that it only calls static methods, so it can be `private static`